src/components/forms/controls/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,207 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../../styling/defs.scss' as bk;
+
+// TODO: copied manually from old Baklava, where should we put this?
+$z-index-date-picker: 100;
+
+// DATEPICKER override react-datepicker styles
+// Note: Sass do not support using suffixes (i.e. &__suffix) when using global selector
+:global(.react-datepicker-popper) {
+  z-index: $z-index-date-picker;
+}
+
+:global(.react-datepicker) {
+  width: 284px;
+  text-align: center;
+  padding: 22px 26px 30px;
+  background-color: bk.$theme-card-background-default;
+  border-radius: 6px;
+  border: 1px solid bk.$theme-card-border-default;
+  font-family: bk.$font-family-body;
+  font-size: bk.$font-size-m;
+  font-weight: bk.$font-weight-regular;
+}
+
+:global(.react-datepicker__triangle),
+:global(.react-datepicker__triangle::before) {
+  display: none;
+}
+
+:global(.react-datepicker__header) {
+  background: transparent;
+  border: 0;
+  color: #081353; // TODO hardcoded color
```

Comment:
Is this even used? Doesn't seem like this color has any effect. Should we remove it?